docs/csharp/programming-guide/arrays/snippets/SingleDimensionArrays.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace arrays
+{
+    public static class SingleDimensionArrays
+    {
+        internal struct SomeType
+        {
+            string thing;
+
```

Comment:
Remove extra line in `struct` definition, but add line after `struct`, before `Examples` method.